src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -91,8 +91,8 @@ azure-synapse-spark==0.2.0
 bcrypt==3.2.0
 certifi==2023.7.22
 cffi==1.15.1
-chardet==3.0.4
-colorama==0.4.4
+chardet==5.2.0
```

Comment:
If Azure CLI has no direct reference to `chardet`? Consider removing it?

Reply:
There is one usage:

https://github.com/azure/azure-cli/blob/85d68d38793fa49afc40257123f3b9f17e891a3d/src/azure-cli/azure/cli/command_modules/appconfig/_kv_helpers.py#L807